Business Finder/MapViewController.h

```diff
@@ -0,0 +1,21 @@
+//
+//  MapViewController.h
+//  Business Finder
+//
+//  Created by Michael Johnson on 2/16/17.
+//  Copyright © 2017 Michael Johnson. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <MapKit/MapKit.h>
+@class MapController;
+@class Business;
+
+@interface MapViewController : UIViewController<MKMapViewDelegate>
+-(void)setBusiness:(Business *)business;
+@property (weak, nonatomic) IBOutlet MKMapView *mapView;
+@property (strong, nonatomic) MapController *controller;
```

Comment:
You have a double-strong relationship between `MapController` and `MapViewController`. I'd consider making one weak so you avoid retain loop.